src/qt/CMakeLists.txt

```diff
@@ -123,7 +123,9 @@ SET(QT_SOURCES
         ${CMAKE_CURRENT_SOURCE_DIR}/pivx/furabstractlistitemdelegate.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/pivx/txviewholder.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/pivx/qtutils.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/pivx/balancebubble.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/pivx/expandablebutton.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/pivx/hoverablebutton.cpp
```

Comment:
seems that `hoverablebutton.cpp` (and .h?) never made it into the repo

Reply:
need to clean it, good. I ended up simplifying the code with an event watcher instead of subclassing the button and overriding the functionality.

Reply:
pushed.